server/commands/pinDestroyer.ts

```diff
@@ -24,31 +23,22 @@ export default async function pinDestroyer({
   user,
   pin,
   ip,
-  transaction: t,
+  transaction,
 }: Props): Promise<Pin> {
-  const transaction = t || (await sequelize.transaction());
```

Comment:
Had to remove the in-command transaction here so that it can be passed in without committing. As a rule going forward all commands will take a `transaction` argument.